src/main/java/com/dev/cinema/util/HashUtil.java

```diff
@@ -0,0 +1,32 @@
+package com.dev.cinema.util;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import org.apache.log4j.Logger;
+
+public class HashUtil {
+    private static Logger logger = Logger.getLogger(HashUtil.class);
+
+    public static byte[] getSalt() {
+        SecureRandom secureRandom = new SecureRandom();
+        byte[] salt = new byte[16];
+        secureRandom.nextBytes(salt);
+        return salt;
+    }
+
+    public static String hashPassword(String password, byte[] salt) {
+        StringBuilder hashedPassword = new StringBuilder();
+        try {
+            MessageDigest messageDigest = MessageDigest.getInstance("SHA-512");
+            messageDigest.update(salt);
+            byte[] digest = messageDigest.digest(password.getBytes());
+            for (byte b : digest) {
+                hashedPassword.append(String.format("%02x", b));
+            }
+        } catch (NoSuchAlgorithmException e) {
+            logger.error(e);
```

Comment:
Why logging? Maybe throw an exception?

Reply:
what is the correct behavior if we will receive the exception here?

Reply:
I don't know the right answer:( 
Maybe, we can add add "throws" to the signature and then handle it in some LoginController?

Reply:
let's throw an exception